index.html

```diff
@@ -160,20 +160,21 @@ <h3 id="h3-lang">
          <h2 class="contact-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
         </div>
         <div>
-         <form class="contact" action="https://formspree.io/f/myylobqj" method="post">
+         <form class="contact" action="https://formspree.io/f/mqkoyzva" method="post">
            <input type="text" name="name" placeholder="Full name" maxlength="30" required>
            <input type="email" name="email" placeholder="Email address" required>
-           <textarea name="message" rows="3" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
-         </form>
+           <textarea name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+           <button type="submit" name="submit" id="contact-button">Get in touch</button>
+        </form>
         </div>      
     </div>
     <section class="bg_final">
     <div class="contact-socials">  
-        <button type="submit" name="submit" id="contact-button">Get in touch</button>
+        
         <div class="contact-me-mail">
-            <h4>jdxelric@gmail.com</h4>
+            <p>jdxelric@gmail.com</p>
         </div> 
-        <ul>
+        <ul class="footer-icons">
```

Comment:
- [ ] Please remove this bar 🔽 , it's actually a phone navigation bar and is not part of the design.

![2023-03-28_231428](https://user-images.githubusercontent.com/104319462/228387761-794398ac-2a23-49d6-824a-967c3f42316b.png)